### DISCUSSION TITLE
Código: 005, Descripción: Factura a dar de alta o subsanar ya registrada en el sistema.

### DISCUSSION THREAD

**Discussion by mmgmanuel** [Barnetik/tbai-php-lib#25] [Q&A]
Buenas tardes,

Tras darnos el error 009 Error de encadenamiento, lo hemos solucionado, pero al volver a enviarlas, nos da el error 005. ¿Cómo podemos solucionarlo?

Leyendo por internet, dicen que usemos el sistema Zuzendu, pero vemos que no lo tenéis implementado. Por si se podría solucionar de otra forma.

Gracias y un saludo.

**Comment by arraintxo** [comment]
Buenas @mmgmanuel ,

lo siento pero no sé si existe alguna otra manera de subsanar el error. Habéis mirado si podéis hacerlo "a mano" a través de los servicios ofrecidos por las diputaciones?
